### PR TITLE
Cult base theme

### LINKED ISSUE #5916: Cult: Create Base Theme
Create a starting point for the theme that gives us a place to iterate from.

### PR DESCRIPTION
**Pull request by MaggieCabrera**
#### Changes proposed in this Pull Request:

This PR creates a a clone of Block Canvas renamed to Cult. Nothing else is done here.

#### Related issue(s):

Closes https://github.com/Automattic/themes/issues/5916

### CODE REVIEW

File: cult/templates/404.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<!-- wp:group {"tagName":"main","layout":{"inherit":true}} -->
+<main class="wp-block-group">
+
+	<!-- wp:pattern {"slug":"cult/404"} /-->
+	<!-- wp:search {"label":""} /-->
+
+</main>
+<!-- /wp:group -->
+
+<!-- wp:template-part {"slug":"footer","tagName":"footer","className":"site-footer-container"} /-->
```
Comment (pbking) [inline cult/templates/404.html]: The source theme (and thus this one) have the 'site-footer-container' class throughout and isn't used for anything.  Suggest removing it from here (and Block Canvas).

**Comment by pbking** [comment]
Other than the `.site-footer-container` class this looks like the start!

File: cult/templates/404.html (hunk 1 above)
Comment (MaggieCabrera) [reply cult/templates/404.html]: yes, let's do that, nice catch!

**Review by pbking — APPROVED** [review]
🚢

**Review by jffng — APPROVED** [review]
Woo!

**Pull request merged**